config/initializers/simple_form_bootstrap.rb

```diff
@@ -45,6 +45,19 @@
   config.input_field_error_class = 'is-invalid'
   config.input_field_valid_class = 'is-valid'
 
+  # Configure layout of date input, time input, and datetime input.
+  SimpleForm::Inputs::DateTimeInput.default_options = {
```

Comment:
What do you think about getting this into the configuration options of simple_form? 

https://github.com/plataformatec/simple_form/blob/10c05896fa7c3be134b1bd5cc728670f383644c6/lib/simple_form.rb#L47-L207

/cc @feliperenan

Reply:
Do you mean changing it to something like `config.date_time_input_default_options`?  Personally, I don't have a preference either way.  Although, I would prefer merging these styling changes sooner and then having a follow-up PR to change to `config.date_time_input_default_options`, as opposed to postponing this PR until a new version of the simple_form gem can be released.

Reply:
@rafaelfranca  I see.  So would it be acceptable if each value were assigned individually?  e.g.

```ruby
SimpleForm::Inputs::DateTimeInput.default_options.tap do |options|
  options[:date_separator] = '<div class="mx-1"></div>'
  options[:time_separator] = '<div class="mx-1">:</div>'
  # etc
end
```

Reply:
Even using `.tap`, I'm not sure if we should do that since `default_options` are for internal use of Simple Form. It isn't a public API.

I'm not sure if there is another way of setting custom options into Simple Form input default, if it doesn't, would make sense create an API for that? 🤔 

cc @rafaelfranca

Reply:
To be honest, I'm not sure if we should add a feature at Simple Form just to support it. I'm more inclined to live with this small issue than maintaining a solution for this specific case.